source/shared/net/desktop-wallet/change-restake-settings.rst

```diff
@@ -26,11 +30,11 @@ Change restake settings (Single-signature account)
 Change restake settings (Multi-signature account)
 =================================================
 
-#. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Update baker restake earnings**.
+#. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Update baker stake**.
 
 #. Select the **Account** whose baker keys you want to update, and then select **Continue**. Only baker accounts are listed.
```

Comment:
This should be updated like the same paragraph for singlesig. (So that it doesn't refer to updating keys)